tests/Feature/BlackoutControllerTest.php

```diff
@@ -123,7 +123,11 @@ public function users_can_create_weekly_blackout()
       'end_date' => Carbon::now()->addWeeks(10)->toDateString()
     ]);
 
-    $blackout = Blackout::factory()->make();
+    $blackout = Blackout::create([
+      'name' => 'test',
+      'start_time' => Carbon::today(),
+      'end_time' => Carbon::today()->addMinute()
+    ]);
```

Comment:
there are also factory states that we should setup instead of making a custom booking for weekly blackout. 

https://laravel.com/docs/8.x/database-testing#applying-states

Reply:
It would be a good chore/bug to add for an issue.